src/Essential.Diagnostics.RollingFileTraceListener/Diagnostics/RollingFileTraceListener.cs

```diff
@@ -218,7 +249,7 @@ protected override void Write(string category, string message, object data)
             }
             else
             {
-                rollingTextWriter.Write(null, message);
+                rollingTextWriter.Write(null, message, NewStreamOnError);
```

Comment:
That way, with lazy creation of the writer, you don't need to pass the same parameter value every time.